crates/ads126x/src/register.rs

```diff
@@ -86,6 +117,50 @@ bitflags! {
     }
 }
 
+impl InterfaceRegister {
+    pub fn default() -> Self {
+        InterfaceRegister::from_bits_truncate(0b0000_0101)
+    }
+
+    pub fn get_crc(&self) -> CrcMode {
+        match self.bits() & 0b0000_0001 {
+            0b00 => CrcMode::Disabled,
+            0b01 => CrcMode::Checksum,
+            0b10 => CrcMode::CRC,
+            _ => unreachable!(),
```

Comment:
0b11 should panic because it's a possible value, not unreachable. Anything that is unreachable (because X & 0b11 will only ever be 0b00..=0b11) is > 0b11. For the values that are possible, it'll make our lives much easier to have a proper panic message should that ever be the result.

See what I've done with the other registers.

Unless you disagree with this. But semantically, unreachable should be impossible mathematically, and 0b11 is totally possible even if it isn't allowed by the ADC. Either way, unreachable will panic if it's hit, but we just lose the nice debug message otherwise.

`unreachable_unchecked!()` is also an option, and this will not panic if it's hit, the compiler literally will not check this and will treat it as impossible, so it's even more performant, but you thus get nothing at all in the event of an error and the program will rip itself to shreds. We *could* use it, but only if we really need this performance.